pootle/core/search/backends/elasticsearch.py

```diff
@@ -26,7 +27,12 @@ def __init__(self, config_name):
         self._create_index_if_missing()
 
     def _server_setup_and_alive(self):
-        return self._es is not None and self._es.ping()
+        if self._es is not None:
```

Comment:
In general, I'd [bail out early](https://philikon.wordpress.com/2010/01/26/flat-is-better-than-nested-bail-out-early/):

```python
if self._es is None:
    return False

try:
    return self._es.ping()
except ConnectionError:
    return False
```